templates/boltzmann/core/prelude.ts

```diff
@@ -47,12 +54,74 @@ if (!process.env.HONEYCOMB_TEAM && process.env.HONEYCOMBIO_TEAM) {
   process.env.HONEYCOMB_TEAM = process.env.HONEYCOMBIO_TEAM
 }
 
-beeline({
-  writeKey: process.env.HONEYCOMB_WRITEKEY,
-  dataset: process.env.HONEYCOMB_DATASET,
-  sampleRate: Number(process.env.HONEYCOMB_SAMPLE_RATE) || 1,
-  serviceName,
-})
+if (!process.env.HONEYCOMB_DATASET && process.env.HONEYCOMBIO_DATASET) {
+  process.env.HONEYCOMB_DATASET = process.env.HONEYCOMBIO_DATASET
+}
+
+let sdk: NodeSDK | null = null
+
+if (
```

Comment:
I'm confused, this section doesn't seem right to me.
Shouldn't we be using the same `isOtel`/`isHoneycomb` from below?
The conditional here doesn't seem right

Reply:
Good call. I started refactoring this in the honeycomb middleware but completely dropped the ball in the prelude. I'm thinking that I should move these functions to the prelude so I can use them in both the prelude and the middleware.

Reply:
Fixed throughout! 🤞